plugins/nextcloud-rule-exclusions-before.conf

```
@@ -1913,6 +1904,26 @@ SecRule REQUEST_FILENAME "@rx /apps/deck/stacks/[0-9]+/reorder$" \
     ver:'nextcloud-rule-exclusions-plugin/1.2.0',\
     setvar:'tx.allowed_methods=%{tx.allowed_methods} PUT'"
 
+# Editing/deleting a comment on a card
+SecRule REQUEST_FILENAME "@rx /ocs/v[0-9\.]+\.php/apps/deck/api/v[0-9\.]+/cards/[0-9]+/comments/[0-9]+$" \
```

`.` is treated as a literal in character classes and doesn't need to be escaped. There are a couple more of these, but I will only suggest it here.
```suggestion
SecRule REQUEST_FILENAME "@rx /ocs/v[0-9.]+\.php/apps/deck/api/v[0-9.]+/cards/[0-9]+/comments/[0-9]+$" \
```